compiler-plugin/src/main/scala/scalapb/package.scala

```diff
@@ -1,11 +1,13 @@
+import com.trueaccord.scalapb.compiler.ProtobufGenerator
 import protocbridge.JvmGenerator
 
 package object scalapb {
   def gen(
     flatPackage: Boolean = false,
     javaConversions: Boolean = false,
     grpc: Boolean = true,
-    singleLineToString: Boolean = false): (JvmGenerator, Seq[String]) =
+    singleLineToString: Boolean = false,
+    collectionType: String = "scala.collection.Seq"): (JvmGenerator, Seq[String]) =
```

Comment:
Let's do a file-level option then. We found that global compiler settings (such as flat_package) that affect the public API of the generated code hurts maintainability issues down the line.